src/components/Home.jsx

```diff
@@ -26,10 +29,22 @@ export default function Home(){
 
     // Handle Rental Form Submission 
     // TODO: Include logic for generating and minting NFT
-    const handleFormSubmit = (formData) => {
+    const handleFormSubmit = async (formData) => {
         const metadata = createNFTMetadata(formData);
         setNftData(metadata);
-        console.log("NFT Metadata created:", metadata);
+        setMintingStatus("Preparing metadata..");
+        console.log("NFT Metadata created:", nftData);
+        metadata.image = nftTestImg
+        setMintingStatus("Minting NFT...");
```

Comment:
Right about here it should check network for testnet.  It tried to mint on main eth net when I used it.